### PR TITLE
Image puller fixes

### PR DESCRIPTION
**Pull request by yhontyk**
### Specify the version of the product this PR applies to.

### CODE REVIEW

File: src/main/pages/che-7/administration-guide/con_image-puller-overview.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // caching-images-for-faster-workspace-start
 
-[id="kubernetes-image-puller-overview_{context}"]
-= Kubernetes Image Puller overview
+[id="image-puller-overview_{context}"]
+= Image Puller overview
```
Comment (rkratky) [inline src/main/pages/che-7/administration-guide/con_image-puller-overview.adoc:4]: I'm not sure if we can do this as the plug-in is called "Kubernetes Image Puller" (https://github.com/che-incubator/kubernetes-image-puller) -- though I'd much prefer if it were called "Image Puller".

Comment (rkratky) [reply src/main/pages/che-7/administration-guide/con_image-puller-overview.adoc:4]: @davidfestal, @ibuziuk -- would it be acceptable to use "Image Puller" to refer to the plug-in?

Side note: Any chance of renaming the plug-in to something platform-agnostic ("Image Puller", for example. Or perhaps "Container Image Puller".)

File: src/main/pages/che-7/administration-guide/proc_deploying-image-puller-using-openshift-templates.adoc [hunk 2]
```diff
@@ -45,11 +31,11 @@ $ cd kubernetes-image-puller
 
 . Create a new OpenShift project to deploy the puller into:
 +
-include::examples/{project-context}-proc_deploying-kubernetes-image-puller-using-openshift-templates_prepare.adoc[]
+include::examples/{project-context}-deploying-kubernetes-image-puller-using-openshift-templates_prepare.adoc[]
```
Comment (rkratky) [inline src/main/pages/che-7/administration-guide/proc_deploying-image-puller-using-openshift-templates.adoc]: A build error (missing include):

```suggestion
include::examples/{project-context}-deploying-image-puller-using-openshift-templates_prepare.adoc[]
```

**Review by rkratky — CHANGES_REQUESTED** [review]

File: src/main/pages/che-7/administration-guide/con_image-puller-overview.adoc (hunk 1 above)
Comment (yhontyk) [reply src/main/pages/che-7/administration-guide/con_image-puller-overview.adoc:4]: I believe hat was a compromise - to use part of the name without mentioning Kubernetes explicitly.

Comment (rkratky) [reply src/main/pages/che-7/administration-guide/con_image-puller-overview.adoc:4]: Ok, thanks, @boczkowska.

I'd still ask @ibuziuk and/or @davidfestal whether renaming the plug-in would be an option.

**Review by rkratky — APPROVED** [review]

**Pull request merged**